canvas/forms/forms.py

```diff
@@ -0,0 +1,37 @@
+from django import forms
+from django.forms import TextInput, CheckboxInput, DateTimeInput, Select
+from canvas.models import CanvasCourse
+from datetime import datetime
+
+
+class CreateEventForm(forms.Form):
+    name = forms.CharField(
+        label="Event Name",
+        max_length=500,
+        widget=TextInput(attrs={'class': 'form-control'})
+    )
+
+    course = forms.ModelChoiceField(
+        label="Course",
+        queryset=CanvasCourse.objects.all(),
+        widget=Select(
+            attrs={'class': 'form-control'}
+        ),
+    )
+    count_for_tokens = forms.BooleanField(
+        label="Does this event count for tokens?",
+        widget=CheckboxInput(attrs={'class': 'form-control'}),
+        required=False
+    )
+
+    start_datetime = forms.DateTimeField(
```

Comment:
I would recommend using a good datetime picker widget. 
You can try this one https://www.jqueryscript.net/time-clock/Lightweight-jQuery-Date-Input-Picker.html

Reply:
Actually that one might not have time in it.
https://xdsoft.net/jqplugins/datetimepicker/
try this one in that case.